Bio/Nexus/Trees.py

```diff
@@ -40,9 +40,7 @@ def __init__(self, taxon=None, branchlength=0.0, support=None, comment=None):
 
 
 class Tree(Nodes.Chain):
-    """Represents a tree using a chain of nodes with on predecessor (=ancestor)
-    and multiple successors (=subclades).
-    """
+    """Represent a tree using a chain of nodes with on predecessor (=ancestor) and multiple successors (=subclades)."""
```

Comment:
Its good that is now a one line summary, but its rather long. Can you perhaps split this into a one line summary, blank line, and a longer line?